webpack/assets/javascripts/react_app/components/SearchBar/search-bar.scss

```diff
@@ -1,15 +1,6 @@
 @import "../AutoComplete/auto-complete.scss";
-@import "~@theforeman/vendor/scss/variables";
 
 .search-bar {
-  .autocomplete-aux {
-    right: 108px;
```

Comment:
This was a hack. I have made the field relative and we are positioning against it, so we do not need to count with the buttons and the css is ok everywhere.